contracts/strategies/allocation/donation-voting-passport-offchain-calc/README.md

```diff
@@ -0,0 +1,57 @@
+Spec: Donation Voting with Passport and Offchain Calculation
+---------------------------------
+
+## Overview 
+There is a category of mechanisms (including Quadratic Funding or QF) allocate a funding pool based upon individual donations to eligible recipients. Essentially, the amount that an individual donates to a project is considered their "vote", which is weighted according to the type of voting calculation formula being used. 
+
+At time of writing (June 2023), most prominent QF formulas are too computationally expensive to be calculated on-chain at a reasonable cost. As a step towards an eventual fully on-chain QF solution, this strategy functions as a hybrid solution: 
+* votes are recorded **on-chain**
+* the allocation is calculated **off-chain**
+* the final distribution is recorded **on-chain**
+
+This strategy also uses Passport as a voter eligibility signal as outlined below. 
+
+## Spec
+### Recipient logic
+In this strategy, recipients must apply and be approved by pool admins. 
+- `registerRecipient` - function for recipient to submit application
+    - if a recipient submits an application that meets all requirements, then the recipient status is set to `Pending`
+    - **Customizable Settings**
+        - pool admin can require recipients to have Allo registry identity in order to apply
+            - if pool admin doesn't require Allo identity and the recipient doesn't have one, then `msg.sender` is used as recipient id
+        - pool admin can include optional and required questions in an application, stored in metadata
+            - recipient must answer questions as specified by pool admin
+        - pool admin can require the recipient to have verified credentials for Twitter or Github
+            - recipient must have those credentials on their recipient id
```

Comment:
can we already proof it on-chain? If not it is an off-chain requirement and we can only check that the metadata is not empty